.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
You can also add
`--format documentation`  
for better readability of the tests format output